app/main.py

```diff
@@ -1,3 +1,57 @@
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> Any:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> Any:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> Any:
```

Comment:
`other` can be `Distance` too. And specify concrete class for return annotation, not just `Any`